### ISSUE TITLE
Yara rules folder

### ISSUE THREAD

**Issue by seifreed** [Rurik/Noriben#11]
Hi I'm trying to use Noriben with a Yara rules folder from https://github.com/Yara-Rules/rules/tree/master/malware

![image](https://cloud.githubusercontent.com/assets/1843119/20965005/3824e95a-bc74-11e6-9e49-73235e84848f.png)

Something wrong?

**Comment by Rurik** [comment]
Hi Marc,
That should work fine. I just tested it here with no issues. It should run the created files through the YARA rules and show results (see screenshot).

However, I did come across an unusual bug in testing, where it wouldn't correctly handle an Internet failure (if you had VT enabled). I'm not sure if that's related, but I did just upload 1.6.4 to correct that. 

![2016-12-07 11_15_22-winxp_malware - vmware workstation](https://cloud.githubusercontent.com/assets/3107827/20976088/6e6779f6-bc6f-11e6-9bd7-d67c28072182.png)

**Comment by seifreed** [comment]
Hi,

 

Tested, all clear!

 

Regards

 

--

Marc Rivero López | @seifreed | ecrime.pro

 

De: Brian Baskin [mailto:notifications@github.com] 
Enviado el: miércoles, 7 de diciembre de 2016 17:23
Para: Rurik/Noriben <Noriben@noreply.github.com>
CC: Marc Rivero López <mriverolopez@gmail.com>; Author <author@noreply.github.com>
Asunto: Re: [Rurik/Noriben] Yara rules folder (#11)

 

Hi Marc,
That should work fine. I just tested it here with no issues. It should run the created files through the YARA rules and show results (see screenshot).

However, I did come across an unusual bug in testing, where it wouldn't correctly handle an Internet failure (if you had VT enabled). I'm not sure if that's related, but I did just upload 1.6.4 to correct that.

 <https://cloud.githubusercontent.com/assets/3107827/20976088/6e6779f6-bc6f-11e6-9bd7-d67c28072182.png>
